README.md

```diff
@@ -1,61 +1,33 @@
 <img alt="FactSet" src="https://www.factset.com/hubfs/Assets/images/factset-logo.svg" height="56" width="290">
 
-# Analytics API Engines R SDK
+# Analytics API V3 R Programming Code Snippets
 
-[![build](https://img.shields.io/github/workflow/status/factset/analyticsapi-engines-r-sdk/CI)](https://github.com/factset/analyticsapi-engines-r-sdk/actions?query=workflow%3ACI)
-[![cran](https://img.shields.io/cran/v/factset.analyticsapi.engines)](https://cran.r-project.org/web/packages/factset.analyticsapi.engines)
-![API version](https://img.shields.io/badge/API-v2-blue)
-[![Apache-2 license](https://img.shields.io/badge/license-Apache2-brightgreen.svg)](https://www.apache.org/licenses/LICENSE-2.0)
-
-Use this library to integrate with FactSet's Analytics APIs. Below APIs are supported by this SDK.
+Use this examples to work with FactSet's Analytics APIs. Below APIs can be found in this Repository.
```

Comment:
`To integrate with FactSet's Analytics APIs. Code snippets are available for below APIs.`

Reply:
done